ksqldb-functional-tests/src/test/resources/query-validation-tests/is_json_string.json

```diff
@@ -0,0 +1,26 @@
+{
+  "comments": [
+    "Tests covering the use of the IS_JSON_STRING function."
+  ],
+  "tests": [
+    {
+      "name": "filter rows that contain valid JSON",
+      "statements": [
+        "CREATE STREAM test (K STRING KEY, maybeJSON STRING) WITH (kafka_topic='test_topic', VALUE_FORMAT='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT K, maybeJSON as JSON FROM test WHERE IS_JSON_STRING(maybeJSON);"
+      ],
+      "inputs": [
```

Comment:
Could we test for `""` as well, as the docs indicated?

Reply:
Good idea! Added here - https://github.com/confluentinc/ksql/pull/8600/files#diff-05c0224326718ad5ceb5dd8d9760fd25ce4c80d1e943d648f5b7f2277de82b84R17.